### PR TITLE
Portfolio Website Desktop Version phase1

### PR DESCRIPTION
**Pull request by mudasiri**
**Portfolio Website Desktop Version phase1**
In this pull request i have optimized my website to fit on desktop devices using the following Responsive Web Technologies:

1. Flex Box
2. CSS Grid
3. Media Queries

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,7 +31,8 @@ <h3><a href="#" class="action-text">LET’S CONNECT</a></h3>
         <div class="grid-container">
```
Comment (Nemwel-Boniface) [inline index.html:31]: - [ ] Great job making your works cards look well in a desktop view. However, note that you are supposed to stick as much as possible to the provided Figma design which includes typography, colors, fonts, etc. Kindly make sure that the works cards appear in the alternating form as they are in the Figma design.

**This is what your design currently looks like:**

image

**This is the desired Figma design:**

image


**Hint:** Give your cards additional classes for those that need to be alternated and use the following one line for the trick! ```flex-direction: row-reverse;```

File: index.html [hunk 2]
```diff
@@ -87,28 +94,32 @@ <h2 class="portfolio-title">Tonic</h2>
...
+        </div>
 
         <ul class="skils-ul">
             <li>
                 <div class="flex-skill-title">
                   <h2>Languages</h2>
                   <i class="fa-sharp fa-solid fa-angle-down"></i>
                 </div>
+           <div>
             <ul class="sub-languages">
```
Comment (Nemwel-Boniface) [inline index.html:123]: - [ ] Kindly ensure that you align the languages correctly in a horizontal direction for the desktop view as it is in the Figma design

**This is how I am seeing your project so far:**

image


**This is the desired Figma design:**

image

File: index.html [hunk 3]
```diff
@@ -151,7 +163,7 @@ <h2>Contact me</h2>
             <input type="text" value="Mudasir">
             <input type="email" name="" id="" value="muda@nonihub.org">
             <textarea name="" id="" cols="30" rows="10" form="contact-form" placeholder="Write your message here"></textarea>
-            <input class="submit-btn" type="submit" value="Get In Touch">
+            <div class="btnholder"><input class="submit-btn" type="submit" value="Get In Touch"></div>
```
Comment (Nemwel-Boniface) [inline index.html:166]: - [ ] Kindly ensure that you add button states that are for interactivity. The states are eg on hover, active etc

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @mudasiri        :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job working on your portfolio website version so far! :clap:
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

**As I was not able to select the piece of code that needs some changes, I will highlight it here with screenshots as aid**

- [ ] KIndly ensure that your desktop version header/ navbar matches the chosen Figma design. It currently does not show the list of the nav items as it is supposed to be like.

**This is how I am seeing the section right now:**

image

**This is the desired Figma design:**

image


_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -13,12 +13,17 @@
...
         <nav>
             <i class="fa-solid fa-bars"></i>
+            <ul class="nav-items">
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
         </nav>
     </header>
     <section class="intro">
```
Comment (bobb-Rob) [inline index.html:23]: - [ ] Well done, you made great effort to make the changes as requested by the previous reviewer. But you still a few things to do, please confirm the proper order of the project as given in the Figma design. I cannot see that UBER project section any longer. THe order:
- [ ] Tonic
- [ ] Multipost
- [ ] facebook 
- [ ] uber

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @mudasiri , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sajid-munawar — APPROVED** [review]
### Hi @mudasiri , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please check my inline comments.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**